### PR TITLE
Adding a random AI player

### PR DESCRIPTION
**Pull request by brantonb**
Doing some research on strategy and it seemed easiest to do a random player first. No unit tests because testing with random is hard. I suppose I could supply an object with a method to get random and pass in known input in the tests. Simple enough that I'm not too concerned about it, though.

### CODE REVIEW

**Comment by brantonb** [comment]
I was too lazy to open an issue and do a different pull request when I discovered subtests, so I threw it into this one.

File: ConnectFour/play_connect_four.py [hunk 1]
```diff
@@ -10,21 +27,18 @@
...
-        col = int(input()) - 1
-        board.drop_piece(piece, col)
-        if piece == Color.red:
-            piece = Color.black
+        board.drop_piece(player.color, player.get_move(board))
+        if player == player1:
+            player = player2
         else:
-            piece = Color.red
+            player = player1
```
Comment (jdharms) [inline ConnectFour/play_connect_four.py]: Consider keeping current and next players in their own variables.  Cleans up your swapping code to just:
    current_player, next_player = next_player, current_player

Comment (brantonb) [inline ConnectFour/play_connect_four.py]: I like that. This syntax is strange to me so I never think to do it.

Comment (jdharms) [inline ConnectFour/play_connect_four.py]: It's pretty crazy.  I remember when I learned it was possible as a freshman and it blew my mind.

**Pull request merged**